src/app/clusters/level-control/level-control.cpp

```diff
@@ -650,16 +652,34 @@ static void moveHandler(uint8_t commandId, uint8_t moveMode, uint8_t rate, uint8
         }
     }
 
-    // If the Rate field is 0xFF, the device should move as fast as it is able.
-    // Otherwise, the rate is in units per second.
+    // If the Rate field is 0xFF, the device should move at the default move rate, if available,
+    // Otherwise, move as fast as possible
```

Comment:
This is a little confusing until one goes and reads the spec, since it's not clear whether "Otherwise" refers to availability of default move rate or to the value of the Rate field.  Maybe s/Otherwise/If there is no default move rate defined/ or so?